system-tests/resources/cli-tools/Dockerfile

```diff
@@ -15,4 +15,7 @@ COPY ./identity-hub-cli.jar .
 
 COPY ./entrypoint.sh .
 
+# health status is determined by the success of the entrypoint script
+HEALTHCHECK --start-period=60s --interval=5s CMD test -f finished.flag && exit 0 || exit 1
```

Comment:
```suggestion
HEALTHCHECK --start-period=60s --interval=5s CMD test -f finished.flag
```

Reply:
Good catch! Simplified the health check as suggested.